src/tests/components/Alerts.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import Alerts from '../../components/Alerts';
+
+const onAlertClick = jest.fn()
+describe('Alerts component', () => {
+  it('renders a single alerts component', () => {
+    const component = shallow(<Alerts variant="test"
+    showAlert={true}
+    onAlertClick={onAlertClick}
+    alertBody="blah blah blah" 
+    alertHeading="blah blah again"
```

Comment:
@BrianSerem, kindly make this more descriptive other than using blah blah blah. Use something like `this is the alert heading`

Reply:
I have made this more descriptive